### PR TITLE
Feature: Inbox chat loading 

### PR DESCRIPTION
**Pull request by maxplt**
# Jira Issue: []

## What?
Add loading to inbox chat while navigate

## Why?
For no bouncing

## How?
Change `fullWidth` prop in `TaskNavigationWrapper` component from:
```
fullWidth={!activeTask}
```
to:
```
fullWidth={!checkedIn}
```

And add `Spinner` to chat when it is no active task
```
{activeTask ? <Chat /> : <Spinner />}
```

## Video
https://user-images.githubusercontent.com/89198006/174747033-469bf5f0-79b1-4634-a2dd-d825797f0514.mov

## Optional checklist
- [ ] Updated `src/changelog.ts`
- [ ] Codescouted
- [ ] Unit tests written
- [ ] Tested locally

### CODE REVIEW

File: src/portals/hope/pages/tasks/TasksPage.tsx [hunk 1]
```diff
@@ -620,6 +620,16 @@ const TasksPage: React.FC = () => {
               }}
             />
           </TaskChatWrapper>
+        ) : (
+          <TaskChatWrapper
+            style={{
+              display: 'flex',
+              alignItems: 'center',
+              justifyContent: 'center',
+            }}
```
Comment (falklinus) [inline src/portals/hope/pages/tasks/TasksPage.tsx:629]: Maybe use an argument to the styled component instead of inline style?

**Review by falklinus — APPROVED** [review]
Noice! I have one suggestion

File: src/portals/hope/pages/tasks/TasksPage.tsx [hunk 2]
```diff
@@ -361,7 +369,7 @@ const TasksPage: React.FC = () => {
         </BackButtonContainer>
       )}
       <Container>
-        <TaskNavigationWrapper fullWidth={!activeTask}>
+        <TaskNavigationWrapper fullWidth={!checkedIn}>
```
Comment (vonElfvin) [inline src/portals/hope/pages/tasks/TasksPage.tsx:372]: Sweet!

**Review by vonElfvin — APPROVED** [review]
💯

**Pull request merged**